conversion_rules_export/csv/aws-cur.csv

```diff
@@ -99,9 +99,9 @@ Provider,1,NA,Not Defined,ASSIGN_STATIC_VALUE,"static_value: AWS
 "
 Region,1,product/region,Not Defined,RENAME_COLUMN,
 ResourceId,1,lineItem/ResourceId,string,RENAME_COLUMN,
-ServiceCategory,1,lineItem/ProductCode,Not Defined,LOOKUP,"destination_value: aws_category
+ServiceCategory,1,lineItem/ProductCode,Not Defined,LOOKUP,"destination_value: ServiceCategory
 reference_dataset_path: conversion_configs/aws/mapping_files/aws_category_mapping.csv
-source_value: product_code
+source_value: line_item_product_code
```

Comment:
For aws-cur(AWS-Cost Usage Report), the source column is lineItem/ProductCode

Reply:
This is the case for CSV format, but not for Parquet.  Does it assume CSV formaat?

Reply:
> This is the case for CSV format, but not for Parquet. Does it assume CSV format?

Yes, for conversion configs **aws-cur**  the rules are according to CSV format where source columns have '/'
For parquet format, conversion configs **aws** is used.

Reply:
> The source column was not changed here, just the "source_value" from the mapping CSV.

Yes, got it. It will work fine for both. But If we want to keep it as per source column standard, I was thinking if it will be better to keep it separate.

Reply:
Okay.

Reply:
Made the change.